classes/Instagram/Core/Proxy.php

```diff
@@ -14,550 +14,598 @@
+	 * @return StdClass Returns the status
+	 * @access public
+	 */
+	public function modifyRelationship( $user_id, $relationship ) {
+		$response = $this->apiCall(
+			'post',
+			$this->api_url . sprintf( '/users/%s/relationship', $user_id ),
+			array( 'action' => $relationship )
+		);
+		return $response->getData();
+	}
+
+	/**
+	 * Add a like form the current user on a media
+	 *
+	 * @param string $media_id Media ID to like
+	 * @return StdClass Returns the status
+	 * @access public
+	 */
+	public function like( $media_id ) {
+		$this->apiCall(
+			'post',
+			$this->api_url . sprintf( '/media/%s/likes', $media_id )
+		);
+	}
+
+	/**
+	 * Delete a like form the current user on a media
+	 *
+	 * @param string $media_id Media ID to unlike
+	 * @return StdClass Returns the status
+	 * @access public
+	 */
+	public function unLike( $media_id ) {
+		$this->apiCall(
+			'delete',
+			$this->api_url . sprintf( '/media/%s/likes', $media_id )
+		);
+	}
+
+	/**
+	 * Add a comment to a media
+	 *
+	 * @param string $media_id Media ID
+	 * @param string $text Comment text
+	 * @return StdClass Returns the status
+	 * @access public
+	 */
+	public function addMediaComment( $media_id, $text ) {
+		$this->apiCall(
+			'post',
+			$this->api_url . sprintf( '/media/%s/comments', $media_id ),
+			array( 'text' => $text )
+		);
+	}
+
+	/**
+	 * Delete a comment from a media
+	 *
+	 * @param string $media_id Media ID
+	 * @param string $comment_id Comment ID to delete
+	 * @return StdClass
+	 * @access public
+	 */
+	public function deleteMediaComment( $media_id, $comment_id ) {
+		$this->apiCall(
+			'delete',
+			$this->api_url . sprintf( '/media/%s/comments/%s', $media_id, $comment_id )
+		);
+	}
+
+	/**
+	 * Make a call to the API
+	 *
+	 * @param string $method HTTP method to use
+	 * @param string $url URL
+	 * @param array $params API parameters
+	 * @param boolean $throw_exception True to throw exceptoins
+	 * @throws APIException, APIAuthException
+	 * @return  \Instagram\Net\ApiResponse Returns teh API response
+	 * @access private
+	 */
+	private function apiCall( $method, $url, array $params = null, $throw_exception = true ){
+
+		$raw_response = $this->client->$method(
+			$url,
+			array(
+				'access_token'  => $this->access_token,
+				'client_id'     => isset( $params['client_id'] ) ? $params['client_id'] : $this->client_id
+			) + (array) $params
+		);
+
+		$response = new \Instagram\Net\ApiResponse( $raw_response );
+
+		if ( !$response->isValid() ) {
+			if ( $throw_exception ) {
+				if ( $response->getErrorType() == 'OAuthAccessTokenException' ) {
+					throw new \Instagram\Core\ApiAuthException( $response->getErrorMessage(), $response->getErrorCode(), $response->getErrorType() );
+				}
+				else {
+					throw new \Instagram\Core\ApiException( $response->getErrorMessage(), $response->getErrorCode(), $response->getErrorType() );
+				}
+			}
+			else {
+				return false;
+			}
+		}
+		return $response;
+	}
```

Comment:
This applies to whole project. Because this is fork of public GitHub project, indentation should follow original project. In this case spaces.